undetected_chromedriver/patcher.py

```diff
@@ -72,6 +62,10 @@ def __init__(
         prefix = "undetected"
         self.user_multi_procs = user_multi_procs
 
+        self.is_old_chromedriver = version_main and version_main <= 114
```

Comment:
@ultrafunkamsterdam Did you get this error?

```
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\site-packages\undetected_chromedriver\__init__.py", line 255, in __init__
    user_multi_procs=user_multi_procs,
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\site-packages\undetected_chromedriver\patcher.py", line 65, in __init__
    self.is_old_chromedriver = version_main and version_main <= 114
TypeError: '<=' not supported between instances of 'str' and 'int'
```

Reply:
Are you passing version_main in as a string? It is supposed to be `None` or an `int`